docs/pages/versions/unversioned/sdk/fingerprint.mdx

```diff
@@ -0,0 +1,148 @@
+---
+title: Fingerprint
+description: A library to generate a fingerprint from a React Native project.
+sourceCodeUrl: 'https://github.com/expo/expo/tree/main/packages/@expo/fingerprint'
+packageName: '@expo/fingerprint'
+platforms: ['node']
+---
+
+import APISection from '~/components/plugins/APISection';
+import { Collapsible } from '~/ui/components/Collapsible';
+import { Terminal } from '~/ui/components/Snippet';
+
+`@expo/fingerprint` provides an API to generate a fingerprint (hash) of your project for use in determining compatibility between the native layer and JavaScript layer of your app. The hash calculation is configurable, but is by default derived from hashing app dependencies, custom native code, native project files, and configuration.
+
+## Installation
+
+`@expo/fingerprint` is included with [expo](expo.mdx) and [expo-updates](updates.mdx) by default.
+
+If you wish to use `@expo/fingerprint` as a standalone package, you can install it using:
+
+<Terminal cmd={['$ npx expo install @expo/fingerprint']} />
+
+## API Usage
+
+```ts
+import * as Fingerprint from '@expo/fingerprint';
+```
+
+## CLI Usage
+
+<Terminal cmd={['$ npx @expo/fingerprint --help']} />
```

Comment:
note that running `npx @expo/fingerprint` when not installed as a direct dependency, could be broken on pnpm isolated mode. currently we don't have a better way to deal with this. just take a note here and add a [task](https://linear.app/expo/issue/ENG-14908/re-export-fingerprint-cli-from-the-expo-package) myself to follow up.